### PR TITLE
fix: empty style object

### PR DESCRIPTION
**Pull request by eliseeborn**

### CODE REVIEW

**Comment by cbt1** [comment]
Could this also been an issue on the other chart where default styling was removed? If not, why is that?

**Comment by eliseeborn** [comment]
> Could this also been an issue on the other chart where default styling was removed? If not, why is that?

I think you mean the button, and in that case the default styling wasn't fully removed in the end.

File: src/object-properties.js [hunk 1]
```diff
@@ -23,7 +25,8 @@ const properties = {
    */
   footnote: '',
   /**
-   * @type {'regular'|'scroll'|'free'}
+   * How the org chart is navigated
```
Comment (cbt1) [inline src/object-properties.js:28]: This is a spec change and is best to do in separate PR. It also looks like a breaking change?

Comment (eliseeborn) [reply src/object-properties.js:28]: It's been this way in the code for the whole time, just documented wrong, but it should be in a different PR nevertheless 👍

Comment (cbt1) [reply src/object-properties.js]: Just remember that it will be breaking change from API Gov perspective, so once you do a new release make sure to those spec changes approved.

**Review by cbt1 — APPROVED** [review]
